modules/ztp-image-based-upgrade-procedure-steps.adoc

```diff
@@ -0,0 +1,191 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-procedure-steps_{context}"]
+= Performing an image-based upgrade on managed clusters at scale in several steps
+
+Upgrade a set of your managed clusters by using the `ImageBasedGroupUpgrade` CR.
+You can append an action after you evaluate the results of the previous steps.
+This allows you to troubleshoot any failed stages throughout the upgrade procedure.
+
+[IMPORTANT]
+====
+Ensure that the action combinations created by appending actions are supported.
+For more information, see _Supported action combinations_.
+====
+
+.Prerequisites
+
+* Log in to the hub cluster as a user with `cluster-admin` privileges.
+* Create policies and `ConfigMap` objects for resources used in the image-based upgrade.
+* Install {lcao} and OADP Operators on all managed clusters through the hub cluster.
+
+.Procedure
+
+. Create a YAML file on the hub cluster that contains the `ImageBasedGroupUpgrade` CR, for example, named `example-group-upgrade.yaml`:
++
+--
+[source,yaml]
+----
+apiVersion: lcm.openshift.io/v1alpha1
+kind: ImageBasedGroupUpgrade
+metadata:
+  name: example-group-upgrade
+  namespace: default
+spec:
+  clusterLabelSelectors: <1>
+  - matchExpressions:
+    - key: name
+      operator: In
+      values:
+      - spoke1
+      - spoke4
+      - spoke6
+  ibuSpec:
+    seedImageRef: <2>
+      image: quay.io/seed/image:4.16.0-rc.1
+      version: 4.16.0-rc.1
+      pullSecretRef:
+        name: "<seed_pull_secret>"
+  extraManifests: <3>
+    - name: example-extra-manifests
+      namespace: openshift-lifecycle-agent
+  oadpContent: <4>
+    - name: oadp-cm
+      namespace: openshift-adp
+  plan: <5>
+  - actions: ["Prep"]
+    rolloutStrategy:
+      maxConcurrency: 2
+      timeout: 2400
+----
+<1> Specify the set of clusters that you want to upgrade.
+<2> Defines the target platform version, the seed image to be used, and the secret required to access the image.
+<3> (Optional) Specify the list of `ConfigMap` resources that contain your custom catalog sources to retain after the upgrade, and your extra manifests to apply to the target cluster that are not part of the seed image.
+<4> Specify the list of `ConfigMap` resources that contain the OADP `Backup` and `Restore` CRs.
+<5> Defines the upgrade plan.
+--
+
+. Apply the `example-group-upgrade.yaml` file by running the following command on the hub cluster:
++
+[source,terminal]
+----
+$ oc apply -f example-group-upgrade.yaml
```

Comment:
Same feedback as in the `modules/ztp-image-based-upgrade-procedure-cancel.adoc` file:
```suggestion
$ oc apply -f <filename>.yaml
```